catalog/transformer/catalog-transformer-csv-queryresponse/src/main/java/ddf/catalog/transformer/csv/CsvMetacardTransformer.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.csv;
+
+import static ddf.catalog.transformer.csv.common.CsvTransformer.createResponse;
+import static ddf.catalog.transformer.csv.common.CsvTransformer.writeMetacardsToCsv;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.BinaryContent;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.transform.CatalogTransformerException;
+import ddf.catalog.transform.MetacardTransformer;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CsvMetacardTransformer implements MetacardTransformer {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(CsvMetacardTransformer.class);
+
+  @Override
+  public BinaryContent transform(Metacard metacard, Map<String, Serializable> arguments)
+      throws CatalogTransformerException {
+
+    if (metacard == null) {
+      LOGGER.debug("Attempted to transform null metacard");
+      throw new CatalogTransformerException("Unable to transform null metacard");
+    }
+
+    Map<String, String> aliases =
+        (Map<String, String>) arguments.getOrDefault("aliases", new HashMap<>());
```

Comment:
❓ Do we want to support all the arguments the QueryResponseTransformer supports?

![image](https://user-images.githubusercontent.com/9013416/53202532-f70a3380-35e3-11e9-833b-561292cdb378.png)

Reply:
This is probably something we would have to discuss at a higher level. 
The reason why I didn't support those 3 arguments is because 
1. It wasn't specified as a requirement
2. Adding the ability for the user to specify their own order, alias, and which fields to hide might be better served as it's own ticket because it's not exactly obvious how to provide those 3 options at least for me.
If you look at the `writeMetacardsToCsv()` method inside the `CsvTransformer` class (which is what is called later on in this Transformer on line 59), it only takes in an `aliasMap` as a parameter but not `attributeOrder` or `hiddenFields`. This would probably take a lot of refactoring to add the ability to specify the three arguments and since it's not a specified requirement, I chose not to support it.